TeamSnapSDK/SDK/TSDKCompletionBlockTypes.h

```diff
@@ -69,11 +69,9 @@ typedef void (^TSDKForumSubscriptionArrayCompletionBlock)(BOOL success, BOOL com
 typedef void (^TSDKBroadcastAlertArrayCompletionBlock)(BOOL success, BOOL complete, NSArray <TSDKBroadcastAlert *> *_Nonnull broadcastAlerts, NSError *_Nullable error);
 typedef void (^TSDKAvailabilityGroupCompletionBlock)(BOOL success, BOOL complete, TSDKAvailabilityGroups *_Nullable availabilities, NSError *_Nullable error);
 
-typedef void (^TSDKSimpleCompletionBlock)(BOOL success, NSError *_Nullable error);
```

Comment:
Where is the duplicate definition for this? I couldn't find it in a search.

Reply:
Line 14

Reply:
ah weird